.gitignore

```diff
@@ -16,8 +16,6 @@ hs_err_pid*
 
 # node_modules
 **/node_modules
-yarn.lock
```

Comment:
@spider-sauce 

It's better to select one technology and ignore the other one. If it's npm, then don't remove the `yarn.lock` file, if it yarn, then don't remove the `package-lock.json` file